Organic/PrefillAdsInjector.php

```diff
@@ -21,6 +21,9 @@ public function __construct( AdsConfig $adsConfig, PrefillConfig $prefillConfig,
     }
 
     public function prefill( $content ) {
+        if ( is_feed() ) {
```

Comment:
It would be better to put this check here
https://github.com/orgnc/wordpress-plugin/blob/master/Organic/Organic.php#L429